data/35e/wizards_of_the_coast/supplement/tome_of_magic/tm_abilities.lst

```diff
@@ -1,12 +1,12 @@
-# CVS $Revision: $ $Author: $ -- Sun Aug 17 15:41:15 2014 -- reformated by prettylst.pl v1.51 (build 24906)
+# CVS $Revision: $ $Author: $ -- Sun Aug 17 15:41:15 2014 -- reformated by prettylst.pl v1.51 (build 24906)SOURCEPAGE:p.10SOURCELONG:Tome of Magic
```

Comment:
CVS comment line now needs a SOURCE set of tags?

Reply:
I'd managed to mistakenly join a couple of lines together, fixing and recommiting now